bbb-learning-dashboard/src/App.js

```diff
@@ -86,20 +93,28 @@ class App extends React.Component {
         .then((json) => {
           if (json.response.returncode === 'SUCCESS') {
             const jsonData = JSON.parse(json.response.data);
-            this.setState({ activitiesJson: jsonData, loading: false });
+            this.setState({
+              activitiesJson: jsonData,
+              loading: false,
+              invalidSessionCount: 0,
+            });
             document.title = `Learning Dashboard - ${jsonData.name}`;
           } else {
             // When meeting is ended the sessionToken stop working, check for new cookies
             this.setDashboardParams();
-            this.setState({ loading: false });
+            this.setState({ loading: false, invalidSessionCount: invalidSessionCount + 1 });
           }
         })
         .catch(() => {
-          this.setState({ loading: false });
+          this.setState({ loading: false, invalidSessionCount: invalidSessionCount + 1 });
         });
     } else {
       this.setState({ loading: false });
     }
+
+    setTimeout(() => {
+      this.fetchActivitiesJson();
+    }, 10000 * (2 ** invalidSessionCount));
```

Comment:
This looks like a large number... What do you use to decide the timeout value?

Reply:
It's 10 seconds. I thought it would have few meeting interactions in less than 10 seconds.. 
and a similar interval is configured in Akka to send a new Json. Do you think it should be lower interval?

Reply:
If the request fails.. next timeout will be 20 seconds, then 40 seconds, 80, and so on...

Reply:
Can `invalidSessionCount` go above, say, 64?

Reply:
Yes.. but once it is exponential we should not worry about this. I mean, the Dashboard should be opened for years to achieve 64.